### ISSUE TITLE
ARM Template Bot app giving error after deployement

### ISSUE THREAD

**Issue by waqarghani** [microsoft/botframework-solutions#1346]
## Project 
**Name**: Virtual Assistant
**Language**:  C# 
## Description
After deploying the resources in Azure portal with ARM Template the Bot service is giving following error: There was an error sending this message to your bot: HTTP status code NotFound. And no message is being sent. But deployement is done and successful without any error.

## To Reproduce
Dowload VSIX for Virtual Assistant. Make a project, open powershell, and use this script .\Deployment\Scripts\deploy.ps1 with appid and apppassword basically what is being recommended in the create your first virtual assistant documentation.

## Expected behavior
A running bot that can Echo back messages.

## Screenshots
![error](https://user-images.githubusercontent.com/29833054/57610175-92ba9880-7570-11e9-86a4-d6e188d50348.jpg)

## Additional context
Thank you for your help. I am using westeurope as deployement resource group location.

**Comment by darrenj** [comment]
Hi - The deployment script doesn't publish your Bot to azure so that might be the root cause. If you hit f5 locally and use the emulator it will likely work just fine. If you right click in VS and publish to the newly created app service the above test in web chat should work.

We've had a little feedback from people that they expect publishing occur we'll look into it.

**Comment by waqarghani** [comment]
@darrenj thank you perfect both the things work .. so i will close it and if for any reason you wanna open it go ahead.

**Comment by BeigeBadger** [comment]
@darrenj the deployment script not publishing also caught me out - I noticed something was up when I downloaded the source for my Virtual Assistant and the folder was looking very empty.

It would be nice to have this highlighted in the documentation, probably in the [Deploy your assistant](https://github.com/microsoft/botframework-solutions/blob/master/docs/tutorials/csharp/virtualassistant.md#deploy-your-assistant) section, then have a note under the [Next Steps](https://github.com/microsoft/botframework-solutions/blob/master/docs/tutorials/csharp/customizeassistant.md#next-steps) section of the Customize your Virtual Assistant guide as this is probably the point where IMHO people will be looking to go live.

I assume the same goes for creating a skill - I haven't got up to that part of the documentation yet. Keep up the great work!
